files-community/Files
# Fix: Fixed layout doesn't update on open tabs when the default layout changes

**Resolved / Related Issues**
   Closes #14580 

**Notes**
The `if` at line 432 is not easily readable. Should I add a comment/split it, or is it fine?

**Validation**
How did you test these changes?
   1. Make sure Files settings > Folders > Sync layout and sorting preferences across directories is enabled
   2. Open two tab
   3. Change the layout in one tab
   4. Switch to the other
